### DISCUSSION TITLE
Welcome to localDataStorage Discussions!

### DISCUSSION THREAD

**Discussion by macmcmeans** [macmcmeans/localDataStorage#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nla-asia** [comment]
Thank you for this library. I was planning to share data across multiple react apps under same domain. But when I update localStorage from one app which is inside iframe, the other app does not get localDataStorage event fired. Is this because of iframe limitation or something wrong?

**Reply by macmcmeans** [reply]
I'm not certain. Did you try subscribing to the Broadcast Channel? Can your apps detect changes this way?
